backend/internal/server/apis/user/user.go

```diff
@@ -123,11 +135,22 @@ func createUserAccount(c *fiber.Ctx) error {
 	}
 
 	// response
-	createdUserResponse := createResponse{
-		Email:  user.Email,
-		UserId: int(u.ID),
-	}
+	// createdUserResponse := createResponse{
+	// 	Email:  user.Email,
+	// 	UserId: int(u.ID),
+	// }
+
+	return responder.OkWithData(c, u)
+
+}
 
-	return responder.OkWithData(c, createdUserResponse)
+func hashPassword(password string) string {
+	key := config.Vars.PasswordKey
+	keyBytes := []byte(key)
 
+	passwordBytes := []byte(password)
+	mac := hmac.New(sha256.New, keyBytes)
+	mac.Write(passwordBytes)
+	hashedPassword := mac.Sum(nil)
+	return base64.StdEncoding.EncodeToString(hashedPassword)
 }
```

Comment:
We need a login function that will hash input and verify with the db

Reply:
In the auth api we have this same function that verifies what is in the db and makes sure they match.

Reply:
Okay awesome